app/ghcup/GHCup/OptParse/Rm.hs

```diff
@@ -175,8 +178,12 @@ rm rmCommand runAppState runLogger = case rmCommand of
       )
       >>= \case
             VRight vi -> do
-              forM_ (_viPostRemove =<< vi) $ \msg ->
-                runLogger $ logInfo msg
+              -- TODO: deduplicate with brickMain
+              cabalStore <- liftIO (runIdentity . CC.cfgStoreDir <$> CC.readConfig)
```

Comment:
This will fail if config file doesn't exist, which might be the case if user installed just ghc and then removed it?
Not sure how common that would be, just pointing out, maybe ignore IOExceptions for that?

Reply:
Fixed.